### PR TITLE
Add data review tool link

### PR DESCRIPTION
**Pull request by kindly**

### CODE REVIEW

File: docs/guidance/evaluating.md [hunk 1]
```diff
@@ -30,11 +30,11 @@ Check whether the project-level data is published using [OC4IDS](../projects/ind
 
     .. markdown::
 
-      You can use a [JSON schema validator](https://json-schema.org/implementations.html#validators) to check that whether your data is in the correct format.
+      You can use the [OC4IDS Data Review Tool](http://standard.open-contracting.org/infrastructure/review) to check that whether your data is in the correct format.
```
Comment (jpmckinney) [inline docs/guidance/evaluating.md]: Add trailing slash to URL. Use HTTPS. Don't merge until URL works.

File: docs/guidance/publishing.md [hunk 2]
```diff
@@ -59,7 +59,7 @@ If you collect detailed data on contracting processes, refer to the [OCDS implem
 
 ### (3) Build your data, systems and processes
 
-Create an OC4IDS JSON file for each project your system has information on and use a [JSON schema validator](https://json-schema.org/implementations.html#validators) to check that the files are structurally correct against OC4IDS.
+Create an OC4IDS JSON file for each project your system has information on and use the [OC4IDS Data Review Tool](http://standard.open-contracting.org/infrastructure/review) to check that the files are structurally correct against OC4IDS.
```
Comment (jpmckinney) [inline docs/guidance/publishing.md]: See my other comment.

File: docs/guidance/evaluating.md (hunk 1 above)
Comment (duncandewhurst) [reply docs/guidance/evaluating.md]: Resolved in https://github.com/open-contracting/infrastructure/pull/144/commits/f15dd288f44ed6410de5637d4103b06d3f436fc3

File: docs/guidance/publishing.md (hunk 2 above)
Comment (duncandewhurst) [reply docs/guidance/publishing.md]: Resolved in https://github.com/open-contracting/infrastructure/pull/144/commits/f15dd288f44ed6410de5637d4103b06d3f436fc3

**Review by jpmckinney — APPROVED** [review]

**Pull request merged**